conf/open-products-facts-realm.json

```diff
@@ -2405,5 +2405,6 @@
   },
   "clientPolicies": {
     "policies": []
-  }
+  },
+  "redisUrl": "${REDIS_URL}"
```

Comment:
Haven't tested if this should be a realm setting or in some other config file, yet. Alas, the PR is untested for now, and basically for me to share the idea.